core/src/main/java/io/apicurio/hub/api/codegen/pre/DocumentPreProcessor.java

```diff
@@ -47,8 +47,12 @@ public class DocumentPreProcessor {
      * @param document
      */
     public void process(Document document) {
-        for (IVisitor proc : processors) {
-            Library.visitTree(document, proc, TraverserDirection.down);
+        for (OpenApi30Visitor proc : processors) {
+            try {
+                Library.visitTree(document, proc, TraverserDirection.down);
+            } catch (Exception e) {
+                System.exit(1);
```

Comment:
Should this be wrapped in a `RuntimeException` and rethrown? Not sure myself... what would the experience be e.g. for the Maven plugin versus dumping the stack trace?

Reply:
Ha ha ha ha!  This was me doing some debugging and needing a line of code to set a breakpoint on.  System.exit was just a quick thing to accomplish that.  Apologies I will remove this.

Reply:
Good catch.  Fixed.  Thanks!